File: DevExpress VCL/ExpressQuantumGrid 6/Demos/Delphi/IssueList/IssueListDepartments.pas
unit IssueListDepartments;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms,
  Dialogs, IssueListForm, ImgList, ComCtrls, ToolWin, ExtCtrls, StdCtrls,
  cxControls, cxContainer, cxEdit, cxTextEdit, cxDBEdit, DBCtrls;

type
  TfrmDepartments = class(TfrmBasic)
    cxDBTextEdit1: TcxDBTextEdit;
    Label1: TLabel;
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  frmDepartments: TfrmDepartments;

implementation

{$R *.dfm}

uses
  IssueListForms, IssueListConst;

initialization

  FormManager.RegisterForm(DepartmentsFormID, TfrmDepartments, 'DEPARTMENTS');


end.

File: DevExpress VCL/ExpressQuantumGrid 7/Demos/CBuilder/IssueList/IssueListDepartments.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "IssueListDepartments.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "IssueListForm"
#pragma link "cxContainer"
#pragma link "cxControls"
#pragma link "cxDBEdit"
#pragma link "cxEdit"
#pragma link "cxTextEdit"
#pragma resource "*.dfm"
TfrmDepartments *frmDepartments;
//---------------------------------------------------------------------------
__fastcall TfrmDepartments::TfrmDepartments(TComponent* Owner)
  : TfrmBasic(Owner)
{
}
//---------------------------------------------------------------------------
